torchvision/prototype/transforms/_augment.py

```diff
@@ -110,15 +114,32 @@ def forward(self, *inpts: Any) -> Any:
         return super().forward(sample)
 
 
+def _mixup_onehotlabel(inpt: features.OneHotLabel, lam: float) -> features.OneHotLabel:
```

Comment:
Nit: shouldn't this be part of the `_BaseMixupCutmix` class?

Reply:
OK, let me include it